labscommunity/protocol-land-remote-helper
# feat: Trigger GitHub sync on push

## Summary

This PR adds a feature to trigger GitHub sync on push by owner or contributor to a Repository on Protocol Land.

## How to Test
1. Install dependencies, build and install remote helper

  ```sh
  pnpm install && pnpm build && npm install -g .
  ```
2. Run PL from PR: [https://github.com/labscommunity/protocol-land/pull/92](https://github.com/labscommunity/protocol-land/pull/92) and create a repo or use already created repo there.
3. Complete test of PL PR: [https://github.com/labscommunity/protocol-land/pull/92](https://github.com/labscommunity/protocol-land/pull/92)
4. Clone the repo, make some changes and push the changes to Protocol Land and check if the repo on GitHub is also updated.
